dev/com.ibm.ws.cdi.mp.context/resources/com/ibm/ws/cdi/mp/context/resources/CDI_MP_CONTEXT.nlsprops

```diff
@@ -0,0 +1,33 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME com.ibm.ws.concurrent/resources/com/ibm/ws/concurrent/resources/CWWKCMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKC
+#COMPONENTNAMEFOR CWWKC MicroProfile Context Propagation
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# This message bundle does not have its own message range, and instead reserves specific error messages from the 
+# com.ibm.ws.concurrent.mp.1.0 message bundle
+
+# do not translate: CDI, CompletionStage
+CWWKC1158.cannot.lazy.enlist.beans=CWWKC1158E: CDI beans used in contextual operations, such as a CompletionStage, must be accessed using a CDI bean operation, such as a public \
+method, before context for the operation is captured. The CDI beans that were not accessed before context was captured are: {0}  
```

Comment:
Would it be possible to state this directly in terms of what the user must do to fix the problem?  For example,
"You must access CDI beans {0} prior to capturing context for the contextual action or task in order for the CDI beans to behave correctly when accessed from the action or task."
The way it is currently written makes it less clear that it is the user's/application's responsibility to take action to fix it.

Reply:
I want to have `{0}` at the end of the message because it can get very long, because ContextualInstance toString() includes lots of information such as what qualifiers are on the bean. Also, a large list of beans could be provided.

I could change the last sentence from:
`The CDI beans that were not accessed before context was captured are: {0}`
to be:
`The following CDI beans must be accessed before capturing context: {0}`
to make it more clear what action the user needs to take

Reply:
That's fine to put the list at the end. However, my comment about writing this more directly and including the word "you" still applies.  Maybe we could combine the two?

"You must access CDI beans prior to capturing context for the contextual action or task in order for the CDI beans to behave correctly when accessed from the action or task. The following CDI beans must be accessed before capturing context: {0}."

It would be good to get the ID reviewer involved here to tell us what we can and cannot say in a message, and they may have some ideas for improving clarity as well.